osbs/api.py

```diff
@@ -166,7 +185,19 @@ def _create_build_config_and_build(self, build_json, namespace):
             # if it doesn't exist, then create it
             logger.debug('build config for %s doesn\'t exist, creating...', build_config_name)
             self.os.create_build_config(json.dumps(build_json), namespace=namespace)
-        return self.os.start_build(build_config_name, namespace=namespace)
+            # under some circumstances, creating a buildConfig might instantiate it, so
+            #  check for an already running build and return it if it exists
+            #  see https://github.com/projectatomic/osbs-client/issues/205
```

Comment:
I was more thinking we'd know before-hand whether we expect a build to be automatically instantiated, i.e. when (a) we're creating a BuildConfig, not updating one, and (b) when there is a trigger present.

Otherwise there might be a race condition: Origin might not have decided to instantiate the build yet, and I don't know whether doing so explicitly will stop it.